kms: key rotation support for vault kms (#14818)
rook/rook: Documentation/Storage-Configuration/Advanced/key-management-system.md

        * `schedule`: the schedule, written in [cron format](https://en.wikipedia.org/wiki/Cron), with which key rotation [CronJob](https://kubernetes.io/docs/concepts/workloads/controllers/cron-jobs/) is created, default value is `"@weekly"`.

!!! note

>     Currently key rotation is supported when the Key Encryption Keys are stored in a Kubernetes Secret and Vault KMS.

Reviewer: ```suggestion
    Currently key rotation is supported when the Key Encryption Keys are stored in Kubernetes Secret or Vault KMS.
```

>     Currently key rotation is supported for the default type, where the Key Encryption Keys are stored in a Kubernetes Secret and Vault KMS.

Reviewer 2: ```suggestion
    Currently key rotation is supported when the Key Encryption Keys are stored in a Kubernetes Secret and Vault KMS.
```

`Vault` is not default type. So I think the text should be changed as above.
